### PR TITLE
Email

### PR DESCRIPTION
**Pull request by linda-tsai**
left to do:
1. embed links to see enlarged email mockups
2. A11y
3. Rich: confirm specs, html file, email on acid
4. Claudia: confirm text

### CODE REVIEW

**Comment by linda-tsai** [comment]
Hello, @parkerpearson , I am trying to merge my pull request from here, but it won't let me. The error message is: "This branch has conflicts that must be resolved". Can you help me with this? 

By the way, I've skimmed through the list under "conflicting files", I didn't work on any of them for this branch. I only focused on the email branch, but the list is showing files related to application header, contextual help, drawer, and inputs. Do you know why that's happening?

Thanks a lot.

File: _includes/header.html [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
+	<link rel="icon" href="/img/origami-logo.png" type="image/x-icon" />
 
 	<link href="{{ site.baseurl }}/main.css" rel="stylesheet" />
 
   <script src="https://use.typekit.net/fix5pzv.js"></script>
 	<script>try{Typekit.load({ async: true });}catch(e){}</script>
 
   <script src="//polyfill.webservices.ft.com/v1/polyfill.min.js"></script>
+	<script src="//cdnjs.cloudflare.com/ajax/libs/velocity/1.2.3/velocity.min.js"></script>
+	<script src="//cdnjs.cloudflare.com/ajax/libs/velocity/1.2.3/velocity.ui.min.jss"></script>
```
Comment (StommePoes) [inline _includes/header.html]: typo, it's not jss

**Comment by StommePoes** [comment]
Hey Linda, it looks like your email work was started on a pretty stale old set of work.

If your email changes are small enough, go grab the latest, freshest version of gh-pages and try making a new email branch from there.

So, before making a new branch when you're in an old branch, git pull origin gh-pages first to make sure you're starting on the latest version of things.

File: _includes/header.html (hunk 1 above)
Comment (StommePoes) [inline _includes/header.html]: n/m I remember Parker fixing this long ago. stale code

**Comment by parkerpearson** [comment]
Hi @linda-tsai I can help get this merged in.

**Comment by parkerpearson** [comment]
Closing in favor of #225

**Pull request closed without merging**